packages/wdio-junit-reporter/src/types.ts

```diff
@@ -49,6 +55,15 @@ export interface JUnitReporterOptions extends Reporters.Options {
      * }
      */
     classNameFormat?: (options: ClassNameFormatOptions) => string
+    /**
+     * Give the ability to override the generated name of a test suite. suiteNameFormat RegExp would be applied to the suite name for normalization.
+     *
+     * @default
+     * testSuiteNameFormatOptions: function (options) {
+     *     return ``${options.suite.title}``
+     * }
+     */
+    testSuiteName?: (options: TestSuiteNameFormatOptions) => string
```

Comment:
```suggestion
    suiteNameFormat?: (options: TestSuiteNameFormatOptions) => string
```

Reply:
suiteNameFormat is already in use by the regex format, that's why I choose this name

Reply:
How about we allow it to be `RegExp | Function` so we have one unified parameter for formatting the suite name?